tests/sentry/integrations/github/test_integration.py

```diff
@@ -99,11 +95,26 @@ def _stub_github(self):
             json={"token": self.access_token, "expires_at": self.expires_at},
         )
 
-        repositories = [
-            {"id": 1296269, "name": "foo", "full_name": "Test-Organization/foo"},
-            {"id": 9876574, "name": "bar", "full_name": "Test-Organization/bar"},
-            {"id": 1276555, "name": "baz", "full_name": "Test-Organization/baz"},
-        ]
+        repositories = {
+            "foo": {
+                "id": 1296269,
+                "name": "foo",
+                "full_name": "Test-Organization/foo",
+                "default_branch": "master",
```

Comment:
I'm adding the default_branch in here since it is included in the get_repositories() call from GH